### PR TITLE
Usn/e2 e add train

### LINKED ISSUE #3807: tests: Create a train
### Description and goal

test e2e with:

1. select project
2. select study
3. select scenario
4. select rolling stock
5. click on map to create a path -> path finding must work
6. compute results
7. check GET display

### Acceptance criteria

all steps must be covered

For more details check the one note named `Rédaction de scenarios de test e2e `

### LINKED ISSUE #4220: test: Running time calculation on a given pathfinding
### Description and goal

- **Start page:** train creation page opens

- **Inputs:**
  - Pathfinding: Quimper - Nice
  - MR: TGV2N2 UM2

- **Steps:**
1. Click on the "Add one or more trains" button

- **Arrival page:** train creation page opens

### Acceptance criteria

- [ ] Update spinner for max. 5sec. 
- [ ] Popup "Train added" at the bottom right of the screen 
- [ ] Train added to list

### PR DESCRIPTION
**Pull request by Uriel-Sautron**
close #3807 
close #4220

### CODE REVIEW

File: front/src/applications/stdcm/views/StdcmRequestModal.jsx [hunk 1]
```diff
@@ -151,7 +151,7 @@ export default function StdcmRequestModal(props) {
             <div className="d-flex flex-column text-center">
               {currentStdcmRequestStatus === STDCM_REQUEST_STATUS.pending && (
                 <>
-                  <img src={rabbit} alt="runnning stdcm" className="w-50"/>
+                  <img src={rabbit} alt="runnning stdcm" className="w-50" />
```
Comment (clarani) [inline front/src/applications/stdcm/views/StdcmRequestModal.jsx:154]: Nice, thank you

File: front/tests/pages/scenario-page-model.ts [hunk 2]
```diff
@@ -138,6 +138,7 @@ class PlaywrightScenarioPage {
 
   async setTrainScheduleName(name: string) {
     await this.getTrainScheduleNameInput.fill(name);
+    await expect(this.getTrainScheduleNameInput).toHaveValue(name);
```
Comment (clarani) [inline front/tests/pages/scenario-page-model.ts:141]: Why did you add this line ?

**Review by Akctarus — APPROVED** [review]
Tested in local and LGTM !

**Review by SharglutDev — APPROVED** [review]
LGTM. Tested and everything works fine, good job :)

**Review by clarani — APPROVED** [review]
LGTM, tested in local ! :heavy_check_mark:

File: front/tests/pages/scenario-page-model.ts (hunk 2 above)
Comment (Uriel-Sautron) [reply front/tests/pages/scenario-page-model.ts:141]: This also allows you to test the input
